bark/runtime/viewer/viewer.py

```diff
@@ -85,6 +86,9 @@ def __init__(self, params=None, **kwargs):
         self.draw_rss_safety_responses = params["Visualization"]["Evaluation"]["DrawRssSafetyResponses",
                                                                                "Flag to specify if visualizating rss safety responses.", False]
 
+        self.invert_rss_polygon = params["Visualization"]["Evaluation"]["InvertRssPolygon",
```

Comment:
I do not understand why a flag for this is required and why the vehicle angle is not being used.